src/system/api/app/include/gunwipc.h

```diff
@@ -11,21 +11,28 @@
 
 /*
     IPC path structure:
-
-        (processId - optional | GNW_ROOT_IPC_PROC_ID_SEPARATOR - only if processId present) OR GNW_PATH_IPC_BROADCAST_PREFIX - optional | pathComponent0 | (GNW_PATH_IPC_COMPONENT_SEPARATOR | pathComponentN) - if needed
+        
+        * Global:
+        pathComponent0 | (GNW_PATH_IPC_COMPONENT_SEPARATOR | pathComponentN) - if needed
+        * Direct:
+        GNW_PATH_IPC_DIRECT_PREFIX | GNW_PATH_IPC_COMPONENT_SEPARATOR | pathComponent0 | (GNW_PATH_IPC_COMPONENT_SEPARATOR | pathComponentN) - if needed
+        * Notification:
+        GNW_PATH_IPC_NOTIFICATION_PREFIX | GNW_PATH_IPC_COMPONENT_SEPARATOR | pathComponent0 | (GNW_PATH_IPC_COMPONENT_SEPARATOR | pathComponentN) - if needed
 
     Correct path examples:
 
         * banana
         * banana/potato/strawberry/pizza
-        * 0:banana
-        * 2:banana/potato
+        * :/banana
+        * :/banana/potato
+        * _/banana
+        * _/banana/potato
     
     Allowed characters: a-z A-Z 0-9
-*/ 
-#define GNW_ROOT_IPC_PROC_ID_SEPARATOR ':'
+*/
+#define GNW_PATH_IPC_NOTIFICATION_PREFIX '_'
```

Comment:
Zwykle używa się symbolu `*` jako symbolu wieloznacznego. Jeśli pełni to rolę ścieżki rozgłoszeniowej, co myślisz o użyciu tego symbolu?

Reply:
W sumie nie pomyślałem o tym, natomiast ścieżki rozgłoszeniowe nie są jeszcze dostępne. Powyższe ścieżki służą do wysyłania komunikatów ale tylko do procesów, których dotyczą (śmierć procesu, który miał utworzone powiązanie w mechanizmie IPC).

Aczkolwiek gwiazdka brzmi dobrze, i dodałem kolejną definicję dla takiego typu komunikatów zgodnie z sugestią.

Jeśli do multicast pasowałby lepiej inny znak niż '_' to chętnie zmienię

Reply:
w takim razie `_` brzmi dobrze